cypress/e2e/nhs-app.cy.js

```diff
@@ -0,0 +1,88 @@
+/// <reference types="cypress" />
+
+const DISCLAIMER_TEXT =
+  "This application is for demo use only. It is not intended for real life use.";
+const STORAGE_AREA_TITLE = "Trauma Tower";
+const BOX_FOUR_TITLE = "Trauma Chest Drain - Box 4";
+// const   LOAD_PAGE =  cy.visit("http://localhost:3000");
+    
+describe("disclaimer pop-up", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000");
+    cy.contains(DISCLAIMER_TEXT);
+  });
+
+  it("clicking accept button disengages disclaimer", () => {
+    cy.contains("Accept").click();
+    cy.contains(DISCLAIMER_TEXT).should("not.exist");
+  });
+
+  it("clicking anywhere on gray screen does nothing", () => {
+    cy.root().click(100, 100);
+    cy.contains(DISCLAIMER_TEXT);
+  });
+});
+
+describe("storage area", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3002");
```

Comment:
This still needs to be port 3000. I meant a const like:
`const SITE_URL = "http://localhost:3000";`

Reply:
All good